### ISSUE TITLE
bad argument #1 to ipairs

### ISSUE THREAD

**Issue by omaranbari** [ray-x/cmp-treesitter#1]
Hi,
I keep getting this error:

```Error detected while processing TextChangedI Autocommands for "*":
E5108: Error executing lua .../packer/start/cmp-treesitter/lua/cmp_treesitter/init.lua:51: bad argument #1 to 'ipairs' (table expected, got nil)```

Thank you for looking into it.
Regards,
Omar

**Comment by ray-x** [comment]
Should been fixed 5d12398007b50d66ebb8e9861693ad949245fdc5

**Comment by omaranbari** [comment]
Thank you!
